templates/base.html

```diff
@@ -0,0 +1,29 @@
+{% load static %}
```

Comment:
Thought it's better for django logic since we load static files before involving it inside head block.

And yup:
In your templates, you can use the {% load static %} template tag to include the static file set up in your settings file. This should be included at the top of every template that uses static files.
https://docs.djangoproject.com/en/4.1/howto/static-files/